src/theme/index.tsx

```diff
@@ -258,6 +268,45 @@ export const ThemedText = {
   DeprecatedError({ error, ...props }: { error: boolean } & TextProps) {
     return <TextWrapper fontWeight={500} color={error ? 'deprecated_red1' : 'deprecated_text2'} {...props} />
   },
+  HeadlineLarge({ error, ...props }: { error: boolean } & TextProps) {
```

Comment:
i think we probably want to stop using these `TextWrapper` components. probably can avoid for now? maybe just add the new fontFamily stuff into theme and that's enough

Reply:
Ya I was curious about this too, but I think design wanted to capture the font size / line height too